src/style.css

```diff
@@ -1 +1,52 @@
 /* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. There is no font-family defined here, which could be an oversight if a custom font is intended for use with the `.stars__star` class.